h/static/scripts/app.coffee

```diff
@@ -1,3 +1,11 @@
+angular = require('angular')
+uuid = require('./vendor/uuid')
+
+# These services are provided in their own angular modules and thus must be
```

Comment:
If I understand you, you mean that having the angular modules these define in our angular module dependencies will break if we don't require these first.

Any reason not to just have all the requires at the top so we don't have to worry about that/